js/canvasFunctions.js

```diff
@@ -29,52 +27,56 @@ function getDist(x, y, x1, y1) {
 }
 
 
-function getCorrectPoints(x, y, x1, y1) {
+function getCorrectPoints(x, y, x1, y1, size) {
```

Comment:
this function gets called from other places in the code without the size (ex. the edgeClice)
so it is better to add a default value to it
- note: same for all the other functions that had its `args` changed

## suggested fix:
`func(...., size=15)`